actors/stage/src/main/java/com/ea/orbit/actors/runtime/Execution.java

```diff
@@ -498,7 +498,7 @@ public void setMessaging(final Messaging messaging)
         Task.allOf(extensions.stream().map(v -> v.stop())).join();
 
         // * cancel all pending messages, and prevents sending new ones
-        //messaging.stop();
+        messaging.stop();
```

Comment:
@DanielSperry wouldn't it make sense to stop the storage provider and other extensions after stopping messaging?